README.md

```diff
@@ -1,9 +1,9 @@
 ## Cats-Collections
 
-[![Build Status](https://api.travis-ci.org/typelevel/cats-collections.png)](https://travis-ci.org/typelevel/cats-collections)
+[![Build Status](https://github.com/typelevel/cats-collections/actions/workflows/ci.yml/badge.svg)](https://github.com/typelevel/cats-collections/actions/workflows/ci.yml?query=branch%3Amaster)
 [![codecov.io](http://codecov.io/github/typelevel/cats-collections/coverage.svg?branch=master)](http://codecov.io/github/typelevel/cats-collections?branch=master)
 [![Maven Central](https://img.shields.io/maven-central/v/org.typelevel/cats-collections-core_2.13.svg)](https://maven-badges.herokuapp.com/maven-central/org.typelevel/cats-collections-core_2.13)
-[![Join the chat at https://gitter.im/typelevel/cats-collections](https://badges.gitter.im/typelevel/cats-collections.svg)](https://gitter.im/typelevel/cats-collections?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Discord](https://img.shields.io/discord/632277896739946517?label=Typelevel%20Discord)](https://discord.gg/XF3CXcMzqD)
```

Comment:
@rossabaker would it be possible to get a channel on discord for cats-collection?

Reply:
I don't see any technical obstacle. I'd rather wonder, if there are enough people involved in `cats-collections` that are or would be there.

Reply:
I don't participate on Discord, but if you decide to pursue a channel, I would ask in `#admin`.